aten/src/ATen/native/quantized/cpu/qconv.cpp

```diff
@@ -1473,6 +1473,7 @@ static at::Tensor _quantized_convolution_onednn(
   // Scales of ONEDNN and PyTorch are reciprocal
   const ideep::scale_t& src_scales = ideep::scale_t(1, 1.0 / act_scale);
 
+#if defined(IDEEP_VERSION_MAJOR) && IDEEP_VERSION_MAJOR>=3 && defined(IDEEP_VERSION_REVISION) && IDEEP_VERSION_REVISION == 0
```

Comment:
What happens with `IDEEP_VERSION_MAJOR < 3`?

Reply:
Thanks for the comment. Add the check failure if `IDEEP_VERSION_MAJOR < 3`.